src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocstyle/InputJavadocStyleDefaultSettingsFive.java

```diff
@@ -0,0 +1,35 @@
+/*
+JavadocStyle
+scope = (default)private
+excludeScope = (default)null
+checkFirstSentence = (default)true
+endOfSentenceFormat = (default)([.?!][ \t\n\r\f<])|([.?!]$)
+checkEmptyJavadoc = (default)false
+checkHtml = (default)true
+tokens = (default)ANNOTATION_DEF, ANNOTATION_FIELD_DEF, CLASS_DEF, CTOR_DEF, \
+         ENUM_CONSTANT_DEF, ENUM_DEF, INTERFACE_DEF, METHOD_DEF, PACKAGE_DEF, \
+         VARIABLE_DEF, RECORD_DEF, COMPACT_CTOR_DEF
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocstyle;
+
+public class InputJavadocStyleDefaultSettingsFive
+{
+   /**
+    * {@return a string}
+    */
+    public String foo() // ok, foo one
```

Comment:
Please remove such ok comments, they don't explain anything.
Remove all others that you introduce in this PR

Reply:
Done.